app/javascript/angular/code/services/sessions_downloader.js

```diff
@@ -15,8 +15,8 @@ angular.module("aircasting").factory("sessionsDownloader", [
       errorCallback
     ) {
       var successCallback = function(data) {
-        preprocessData(data, sessions, params);
-        refreshSessionsCallback();
+        preprocessData(data.sessions, sessions, params);
```

Comment:
This change is correct? it seems strange done by itself w/o changing `preprocessData`

Reply:
Yes, cause the structure fo `data` changed. Before it just contained a list of sessions and now it's like this:
```
{
        sessions: [],
        availableSessionsCount: 0
      }
```